### PR TITLE
Turn to AprilTag and comp button bindings

### PR DESCRIPTION
**Pull request by zabzzz**

### CODE REVIEW

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -147,17 +147,15 @@ private void configureButtonBindings() {
...
+    m_driverController.b().whileTrue(new TurnToAngle(-90));
     m_driverController.a().whileTrue(new AlignScore());
 
-    m_coDriverController.rightStick().whileTrue(new setXCommand());
     m_coDriverController.rightTrigger().onTrue(new SimpleScoreNote(ManipulatorConstants.PIVOT_AMP_GOAL, ManipulatorConstants.SCORE_AMP_SPEED, ManipulatorConstants.PIVOT_SHOOTER_THRESHOLD));  
     m_coDriverController.rightBumper().onTrue(new SimpleScoreNote(ManipulatorConstants.PIVOT_FAR_SCORE, ManipulatorConstants.SCORE_SIMPLE_RPM, ManipulatorConstants.PIVOT_SHOOTER_THRESHOLD));
     m_coDriverController.leftBumper().onTrue(new SimpleScoreNote(ManipulatorConstants.PIVOT_CLOSE_SCORE, ManipulatorConstants.SCORE_SIMPLE_RPM, ManipulatorConstants.PIVOT_INTAKE_THRESHOLD));
     m_coDriverController.y().whileTrue(new ScoreReset());
     m_coDriverController.a().whileTrue(new ShootPrep(ManipulatorConstants.PIVOT_CLOSE_SCORE, ManipulatorConstants.SCORE_AMP_SPEED, ManipulatorConstants.PIVOT_SHOOTER_THRESHOLD));
-    m_coDriverController.b().whileTrue(new Shooter(ManipulatorConstants.SCORE_SIMPLE_RPM));
```
Comment (minhd-vu) [inline src/main/java/frc/robot/RobotContainer.java:160]: we're removing the rev shooter button?

Comment (minhd-vu) [reply src/main/java/frc/robot/RobotContainer.java:160]: nvm i see the shoot prep

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -147,17 +147,15 @@ private void configureButtonBindings() {
     m_driverController.leftBumper().onTrue(new PivotPID(ManipulatorConstants.PIVOT_AMP_GOAL, ManipulatorConstants.PIVOT_INTAKE_THRESHOLD));
     m_driverController.rightBumper().whileTrue(new PivotDown(0.5, -0.1));
     m_driverController.start().whileTrue(new ResetRobotHeading());
-    m_driverController.x().whileTrue(new AlignScore(90));
-    m_driverController.b().whileTrue(new AlignScore(-90));
+    m_driverController.x().whileTrue(new TurnToAngle(90));
+    m_driverController.b().whileTrue(new TurnToAngle(-90));
```
Comment (minhd-vu) [inline src/main/java/frc/robot/RobotContainer.java:151]: are these working?

File: src/main/java/frc/robot/commands/drivetrain/vision/AlignScore.java [hunk 3]
```diff
@@ -4,61 +4,48 @@
...
-    }
-
+    angle = RobotContainer.limeSubsystem.getAngleOffset();
     if (Math.abs(angle) > VisionConstants.threshold){
-      RobotContainer.swerveDrive.turnToAngle(rightX);
+      RobotContainer.swerveDrive.drive(
+        RobotContainer.getDriverLeftY(),
+        RobotContainer.getDriverLeftX(),
+        -(VisionConstants.simpleAlignYInput * (angle/100.0)),
+                true, true);
```
Comment (minhd-vu) [inline src/main/java/frc/robot/commands/drivetrain/vision/AlignScore.java]:
```suggestion
      RobotContainer.swerveDrive.drive(
        RobotContainer.getDriverLeftY(),
        RobotContainer.getDriverLeftX(),
        -VisionConstants.simpleAlignYInput * angle / 100.0,
        true, true);
```

File: src/main/java/frc/robot/commands/drivetrain/vision/AlignScore.java [hunk 4]
```diff
@@ -4,61 +4,48 @@
...
     } else {
       RobotContainer.swerveDrive.drive(
         RobotContainer.getDriverLeftY(),
         RobotContainer.getDriverLeftX(),
         RobotContainer.getDriverRightX(),
-            true, true);
+                true, true);
```
Comment (minhd-vu) [inline src/main/java/frc/robot/commands/drivetrain/vision/AlignScore.java]:
```suggestion
        true, true);
```

File: src/main/java/frc/robot/commands/drivetrain/vision/TurnToAngle.java [hunk 5]
```diff
@@ -7,28 +7,33 @@
...
         output -> RobotContainer.swerveDrive.turnToAngle(output));
     addRequirements(RobotContainer.swerveDrive);
+    turn = RobotContainer.swerveDrive.getCorrectAngleTarget(angle);
   }
 
   // Returns true when the command should end.
   @Override
   public boolean isFinished() {
-    return false;
+    return Math.abs((RobotContainer.swerveDrive.getHeading().getDegrees()-turn)) < VisionConstants.threshold;
```
Comment (minhd-vu) [inline src/main/java/frc/robot/commands/drivetrain/vision/TurnToAngle.java]:
```suggestion
    return Math.abs(RobotContainer.swerveDrive.getHeading().getDegrees() - turn) < VisionConstants.threshold;
```

File: src/main/java/frc/robot/RobotContainer.java (hunk 2 above)
Comment (zabzzz) [reply src/main/java/frc/robot/RobotContainer.java:151]: more tuning required

**Review by minhd-vu — APPROVED** [review]

**Pull request merged**